### ISSUE TITLE
[begin-functions-app] npx hydrate errors with missing AWS_REGION in enviroment

### ISSUE THREAD

**Issue by kmelve** [smallwins/begin-issues#1]
**Expected behavior**
Expected `npx hydrate` to do its thing.

**Description of bug**
`npx hydrate` errors with `Error missing AWS_REGION in environment` 

**Steps to reproduce**

```sh
~/Sites
❯ git clone git@github.com:kmelve/begin-functions-app.git
Cloning into 'begin-functions-app'...
remote: Counting objects: 52, done.
remote: Compressing objects: 100% (41/41), done.
remote: Total 52 (delta 14), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (52/52), 34.73 KiB | 362.00 KiB/s, done.
Resolving deltas: 100% (14/14), done.

~/Sites
❯ cd begin-functions-app

~/Sites/begin-functions-app master
❯ npm i

> leveldown@4.0.1 install /Users/knutmelvaer/Sites/begin-functions-app/node_modules/leveldown
> prebuild-install || node-gyp rebuild

npm WARN begin-functions-app@1.0.0 No repository field.
npm WARN begin-functions-app@1.0.0 No license field.

added 382 packages from 348 contributors in 10.184s
[!] 1 vulnerability found [777 packages audited]
    Severity: 1 Low
    Run `npm audit` for more detail


~/Sites/begin-functions-app master 11s
❯ npx hydrate
Error missing AWS_REGION in environment
```

**Shell**
 - OS: macOS
 - Shell: ZSH
 - Node: v9.11.1

**Comment by kmelve** [comment]
I was able to overcome this issue by looking up the [arc documentation](https://arc.codes/quickstart) and running:

```
❯ export AWS_REGION=eu-west-1; export AWS_PROFILE=work; npx hydrate
```

**Comment by kmelve** [comment]
The local server also errs because of the overlying issue. I guess it's because I haven't an AWS profile named `work` in my local ~/.aws folder. The documentation makes no mention that this is a prerequisite (also, it shouldn't be since Begin is a layer between me and AWS lambdas?). 

```
> begin-functions-app@1.0.0 start /Users/knutmelvaer/Sites/begin-functions-app
> NODE_ENV=testing ARC_LOCAL=1 npx sandbox


        ARC_APP_NAME test-app
          AWS_REGION eu-west-1
         AWS_PROFILE work

                Node 9.11.1
                 npm 6.0.1
                .arc 3.1.18

✓ @tables created in local database
✓ src/shared copied to lambda node_modules/@architect/shared
✓ .arc copied to lambda node_modules/@architect/shared/.arc
✓ @events pub/sub ready on local event bus

 @html
 get / ................................. get-index

 ✓ Started HTTP "server" @  http://localhost:3333
{ Error: connect ETIMEDOUT 169.254.169.254:80
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1174:14)
  message: 'Missing credentials in config',
  errno: 'ETIMEDOUT',
  code: 'CredentialsError',
  syscall: 'connect',
  address: '169.254.169.254',
  port: 80,
  time: 2018-07-22T08:45:00.481Z,
  originalError:
   { message: 'Could not load credentials from any providers',
     errno: 'ETIMEDOUT',
     code: 'CredentialsError',
     syscall: 'connect',
     address: '169.254.169.254',
     port: 80,
     time: 2018-07-22T08:45:00.481Z,
     originalError:
      { errno: 'ETIMEDOUT',
        code: 'ETIMEDOUT',
        syscall: 'connect',
        address: '169.254.169.254',
        port: 80,
        message: 'connect ETIMEDOUT 169.254.169.254:80' } } }
/Users/knutmelvaer/Sites/begin-functions-app/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

Error: connect ETIMEDOUT 169.254.169.254:80
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1174:14)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! begin-functions-app@1.0.0 start: `NODE_ENV=testing ARC_LOCAL=1 npx sandbox`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the begin-functions-app@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/knutmelvaer/.npm/_logs/2018-07-22T08_45_00_518Z-debug.log
```

**Comment by ryanblock** [comment]
Oy, sorry about that. Kind of a crappy way to say hello.

We're aware of the issue, though, and it's a priority bug fix for us.

Please stand by!

**Comment by brianleroux** [comment]
Didn't mean to close this. Fixes the `npx hydrate` case but the local webserver not yet. Digging in!

**Comment by brianleroux** [comment]
thx for helping us dig into that @kmelve / should be fixed in `@architect/workflows@3.2.15` 🙏
